### ISSUE TITLE
 ddPrecision: 8 is not working(It only takes 6)

### ISSUE THREAD

**Issue by dipeshhkc** [nerdstep/react-coordinate-input#37]
import CoordinateInput from 'react-coordinate-input';

```
 <CoordinateInput
              //name="dms"
                onChange={(e, { dd, dms, dmsArray }) => {
                      //  console.log(dms, dd, dmsArray);
               }}
               ddPrecision={8}  />
```

After setting ddPrecision to 8,it still takes max of 6 decimals

**Comment by dipesh429** [comment]
import CoordinateInput from 'react-coordinate-input';

```
 <CoordinateInput
                      //name="dms"
                      //value={values.dms || ''}
                      onChange={(e, { dd, dms, dmsArray }) => {
                      //  console.log(dms, dd, dmsArray);
                     }}
                      ddPrecision={8}
                   
                    />
```

**Comment by nerdstep** [comment]
@dipesh429 thank you for opening the issue. I'm a little busy at the moment but will take a look at this as soon as I can.

**Comment by nerdstep** [comment]
@dipesh429 sorry this has taken so long -- I just released a new `1.0.0-beta` version of the component.  This should resolve the issue that you described.
